vagrant/base.sh

```diff
@@ -64,15 +64,20 @@ get_kafka() {
 
     kafka_dir=/opt/kafka-$version
     url=https://s3-us-west-2.amazonaws.com/kafka-packages-$version/kafka_$scala_version-$version.tgz
+    # the .tgz above does not include the streams test jar hence we need to get it separately
+    url_streams_test=https://s3-us-west-2.amazonaws.com/kafka-packages-$version/kafka-streams-$version-test.jar
```

Comment:
This seems fine. We might want to just add a new target to the build which includes all the test jars in a tgz as well, e.g. a `releaseTestTarGzAll` target or something. iirc there are some other test dependencies that are needed that are worked around in a different way in the tests and this would only help with newer versions, but it'd ensure that from now on we have a way to get everything in 1 download.